packages/ui/test/proposals/modals/AddNewProposalModal.test.tsx

```diff
@@ -677,16 +677,7 @@ describe('UI: AddNewProposalModal', () => {
         })
       })
 
-      describe('Type - Fill Working Group Lead Opening', () => {
-        beforeAll(() => {
-          seedWorkingGroups(server.server)
-          seedOpeningStatuses(server.server)
-          seedOpenings(server.server)
-          seedApplications(server.server)
-          seedWorkers(server.server)
-          updateWorkingGroups(server.server)
-        })
```

Comment:
@stepien4soft FYI: this is not needed. See top of the file –> we already seed everything before the tests start. THe mock server is not restarted between tests to speed up execution.